README.md

```diff
@@ -2,7 +2,13 @@
 
 Kotlin Multiplatform project repository template to set up all projects at [Nimble](https://nimblehq.co/)
```

Comment:
```suggestion
Kotlin Multi-platform Mobile (KMM) template to set up all KMM projects at [Nimble](https://nimblehq.co/)
```

Reply:
@lydiasama Multiplatform is a single word https://kotlinlang.org/docs/multiplatform.html

```suggestion
Kotlin Multiplatform Mobile (KMM) template to set up all KMM projects at [Nimble](https://nimblehq.co/)
```